include/exadg/incompressible_navier_stokes/spatial_discretization/spatial_operator_base.cpp

```diff
@@ -1577,6 +1568,17 @@ SpatialOperatorBase<dim, Number>::setup_projection_solver()
                                        projection_operator->get_dof_index(),
                                        projection_operator->get_quad_index());
     }
+    else if(param.preconditioner_projection == PreconditionerProjection::PointJacobi)
```

Comment:
Why is this showing up here? You just added a new elementwise preconditioner since you were at it?

Reply:
yes. I also commented in this commit here https://github.com/exadg/exadg/pull/599#issuecomment-1821040787